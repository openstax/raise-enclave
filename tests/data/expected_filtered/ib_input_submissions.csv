user_uuid,course_id,impression_id,timestamp,content_id,input_content_id,variant,response
d35a986b-a7ad-4c61-8e76-205eccaf9a06,3,3fa85f64-5717-4562-b3fc-2c963f66afa6,0,55585f64-5717-4562-b3fc-2c963f66afa6,99995f64-5717-4562-b3fc-2c963f66afa6,second,100