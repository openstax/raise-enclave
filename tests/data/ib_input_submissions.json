[
    {
        "user_uuid": "265fcd32-9abc-4348-ad52-29995d8142c0",
        "course_id": 0,
        "impression_id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
        "source_uri": "https://localhost:8000",
        "timestamp": 0,
        "eventname": "ib_input_submitted_v1",
        "content_id": "55585f64-5717-4562-b3fc-2c963f66afa6",
        "variant": "main",
        "response": "ASDF",
        "input_content_id": "88885f64-5717-4562-b3fc-2c963f66afa6"
    },
    {
        "user_uuid": "265fcd32-9abc-4348-ad52-29995d8142c0",
        "course_id": 0,
        "impression_id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
        "source_uri": "https://localhost:8000",
        "timestamp": 0,
        "eventname": "ib_input_submitted_v1",
        "content_id": "55585f64-5717-4562-b3fc-2c963f66afa6",
        "variant": "second",
        "response": "100",
        "input_content_id": "99995f64-5717-4562-b3fc-2c963f66afa6"
    }
]