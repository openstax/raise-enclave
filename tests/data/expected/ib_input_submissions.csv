user_uuid,course_id,impression_id,timestamp,content_id,input_content_id,variant,response
265fcd32-9abc-4348-ad52-29995d8142c0,0,3fa85f64-5717-4562-b3fc-2c963f66afa6,0,55585f64-5717-4562-b3fc-2c963f66afa6,88885f64-5717-4562-b3fc-2c963f66afa6,main,ASDF
265fcd32-9abc-4348-ad52-29995d8142c0,0,3fa85f64-5717-4562-b3fc-2c963f66afa6,0,55585f64-5717-4562-b3fc-2c963f66afa6,99995f64-5717-4562-b3fc-2c963f66afa6,second,100